ubuntu/python/README.md

```diff
@@ -0,0 +1,15 @@
+# Python Container
```

Comment:
Could we document which DBR versions this dockerfile is expected to work with?

Reply:
Just checked, seems like 7.x does not work with these package versions. Is it acceptable to say it's supported for 8+?

Reply:
nevermind, it was due to the ipython version being too high. Downgraded it, added a comment, and a clarification here in the README that this recipe supports 7.3 LTS and above.

Reply:
Great, thanks for confirming!